addons/accessory/stringtable.xml

```diff
@@ -12,6 +13,7 @@
         </Key>
         <Key ID="STR_CBA_accessory_railNext">
             <English>Next rail item state</English>
+            <German>Nächster Zustand Schienenaufsatz</German>
```

Comment:
Could "Modus" be used here? Or atleast "Nächster Schienenaufsatz Zustant"?
This sounds very weird.

Reply:
STR_CBA_accessory_railNext_tooltip also uses "Modus"

Reply:
+1 für mode/Modus.
Unabhängig davon: entweder "Zustand des Aufsatzes" oder "Aufsatzzustand". Nix mit Leerzeichen.

Reply:
+1 for "Modus"

Reply:
>STR_CBA_accessory_railNext_tooltip also uses "Modus"

ENG uses "state" in STR_CBA_accessory_railNext and "mode" in STR_CBA_accessory_railNext_tooltip.
state - Zustand, mode - Modus.

>entweder "Zustand des Aufsatzes" oder "Aufsatzzustand". Nix mit Leerzeichen.

They're not compound nouns how I used them. Just two successive nouns. Exactly like in the English original.
ENG is not in possessive form either, so why would I change it to genitive in German? It uses the nominative case, so I did the same.